backend/src/main/java/corea/feedback/service/UserFeedbackService.java

```diff
@@ -0,0 +1,83 @@
+package corea.feedback.service;
+
+import corea.exception.CoreaException;
+import corea.exception.ExceptionType;
+import corea.feedback.dto.FeedbackResponse;
+import corea.feedback.dto.FeedbacksResponse;
+import corea.feedback.dto.UserFeedbackResponse;
+import corea.feedback.repository.SocialFeedbackRepository;
+import corea.feedback.repository.DevelopFeedbackRepository;
+import corea.room.domain.Room;
+import corea.room.repository.RoomRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static corea.global.util.MapHandler.extractDistinctKeyStreams;
+import static corea.global.util.NullHandler.emptyListIfNull;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class UserFeedbackService {
+
+    private final RoomRepository roomRepository;
+    private final DevelopFeedbackRepository developFeedbackRepository;
+    private final SocialFeedbackRepository socialFeedbackRepository;
+
+    public UserFeedbackResponse getDeliveredFeedback(long id) {
+        Map<Long, List<FeedbackResponse>> reviewerToRevieweeMap = getDeliveredReviewerToRevieweeFeedback(id);
+        Map<Long, List<FeedbackResponse>> revieweeToReviewerMap = getDeliveredRevieweeToReviewerFeedback(id);
```

Comment:
```suggestion
        Map<Long, List<FeedbackResponse>> developFeedbackMap = getDeliveredDevelopFeedback(id);
        Map<Long, List<FeedbackResponse>> socilaFeedbackMap = getDeliveredSocialFeedback(id);
```

해당 변수명도 바꿔보는 건 어떨까용? 
리뷰어와 리뷰이라는 단어의 원형이 남아있지 않은 상황에서 해당 도메인 용어를 사용하는 것이 다소 어색하게 느껴집니다.